packages/cloudapi-gql/src/schema/resolvers.js

```diff
@@ -151,8 +151,32 @@ const resolvers = {
       compression ? compression.toUpperCase() : compression
   },
   Mutation: {
+    stopMachine: (root, { id }) =>
```

Comment:
Can you leave spaces between these muytations ? It makes it kind of hard to read :(

Reply:
done ;)